.github/workflows/tests.yml

```
@@ -38,8 +38,21 @@ jobs:
 
       - run: brew test-bot --only-tap-syntax
 
+      - name: Dump merge_group env
+        if: github.event_name != 'pull_request'
+        run: |
+          echo '```' >> "$GITHUB_STEP_SUMMARY"
+          env | tee -a "$GITHUB_STEP_SUMMARY"
```

```suggestion
          env | grep -Eiv 'token|key|secret' | tee -a "$GITHUB_STEP_SUMMARY"
```

given this is publicly visible